Add files via upload

spencb-bit opened the pull request:
-adding ell_accessors_test.cpp, needs to be linked/integrated with whatever ELL class architecture is made.

spencb-bit committed:
Add files via upload

-adding ell_accessors_test.cpp, needs to be linked/integrated with whatever ELL class architecture is made.

cfruble commented on tests/ell_accessors_test.cpp:
```diff
@@ -0,0 +1,301 @@
+// Main
+
+// ==========================================================
+
```
I don't think that you used the test suite functions that we defined in class.  It would be better if you used the `TEST_CASE`, `TEST_SUITE` and `RUN_SUITE` methods that were defined in `unit_test_framework.hpp`

cfruble commented on tests/ell_accessors_test.cpp:
```diff
@@ -0,0 +1,301 @@
+
+
+// ==========================================================
+
```
I think I mentioned this below, but these test utilities already exist in the `example_unit_test_framework.hpp`

cfruble reviewed (commented):
Overall, I everything looks really good.  I think all of the tests done are appropriate.  Just a few small things.  First, I think that at the top you should use `#import <SpMV.hpp>` to get access to the rest of the build.  Second, I would recommend using the example unit test framework that was already defined.

spencb-bit committed:
Add files via upload

modified ell_accessors_test to align with existing testing framework provided as per comments.
